tox.ini

```diff
@@ -25,20 +25,23 @@ pass_env =
     TERM
     USER
 set_env =
+    COVERAGE_FILE = {env:COVERAGE_FILE:{envdir}/.coverage.{envname}}
     COVERAGE_PROCESS_START = {toxinidir}/pyproject.toml
     FORCE_COLOR = 1
     PIP_CONSTRAINT = {toxinidir}/.config/constraints.txt
     PRE_COMMIT_COLOR = always
     TERM = xterm-256color
 commands_pre =
-    sh -c "rm -f .coverage* coverage.xml 2>/dev/null || true"
+    python -c 'import pathlib; pathlib.Path("{env_site_packages_dir}/cov.pth").write_text("import coverage; coverage.process_startup()")'
```

Comment:
@cidrblock Is that for debug or it enables something?

Reply:
It writes a `.pth` file to the site packages directory, that every python process executed later will run. It's needed for `pytest-xdist`, each fork will pick it up and there fore enable coverage for it's process.

It allows us to not need `coverage_enable_subprocess` (or `pytest-cov`).  Some related information can be found here:

https://coverage.readthedocs.io/en/latest/subprocess.html#configuring-python-for-sub-process-measurement

That def looks for a ENV variable called "COVERAGE_PROCESS_START" which point the process to the configuration file: https://github.com/nedbat/coveragepy/blob/60a5d65eade29adbbb16f22a333dab1ca2e99993/coverage/control.py#L1340

Since it's in the tox virtual environment and only in the tox virtual environment, I'm assuming we will always have coverage installed and the ENV set.